.gitignore

```diff
@@ -99,16 +99,21 @@ oprofile_data/
 CMakeSettings.json
 /ci-artifacts/
 /out/
-/pcsx2-winrt/resources/cheats_ws.zip
-/pcsx2-winrt/resources/cheats_ni.zip
+
+# WINRT/UWP Files
+/pcsx2-winrt/resources/patches.zip
 /bin
 /AppPackages/pcsx2-winrt
 /packages
 /pcsx2-winrt/resources
 /pcsx2-winrt/Generated Files/winrt
 /pcsx2-winrt/BundleArtifacts
-/pcsx2-winrt/readme.txt
+/pcsx2-winrt/README.md
 /pcsx2-winrt/pcsx2-winrt_TemporaryKey.pfx
 /Search Results.txt
 /findstr_CLI.bat
 /findstr_CLI(1).bat
+
+# GitHub Desktop Stupidity
+PCSX2_qt.sln
+/pcsx2-winrt/pcsx2-winrt.vcxproj.filters
```

Comment:
These really shouldn't be ignored by git. There's legitimate reasons to want to change them.

Reply:
Files were removed from the list.